docs/release-notes/Percona-Everest-1.7.0-(2025-05-29).md

```diff
@@ -0,0 +1,218 @@
+    | **9.**|[Known limitations](https://docs.percona.com/everest/release-notes/Percona-Everest-1.6.0-%282025-04-16%29.html#known-limitation)|Discover all the known limitations in Percona Everest 1.7.0|
+
+
+## 🌟 Release highlights
+
+=== "☁️ GKE Autopilot clusters"
+
+    ### Deploy Percona Everest on GKE Autopilot
+
+     You can now install Percona Everest on Google Kubernetes Engine (GKE) Autopilot clusters. GKE Autopilot provides a fully managed Kubernetes environment, where Google automatically handles node provisioning, scaling, and security. 
+
+    Here are the benefits of this enhancement:
+
+    - Optimized resource allocation
+    - Streamlined operations
+    - Reduced operational overhead
+    - Simplified cluster management
+    - Automated scaling and built-in security policies—leveraging Percona Everest’s database management capabilities.
+
+
+    📚 Learn more about GKE Autopilot in the official [documentation](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-overview){:target="_blank"}.
+
+===  "⎈ Pod scheduling policies"
+
+    ### Optimized Pod scheduling policies in Percona Everest
+    
+    We are thrilled to introduce Pod scheduling policies for Percona Everest. This feature supports database workload scheduling on Kubernetes by allowing users to define Pod scheduling policies, including Kubernetes Affinity and Anti-Affinity rules. These policies optimize performance and enhance system resilience and ensure that your resources are utilized to their fullest potential.
+    
+    Pod scheduling policy is a **preset** that includes a set of Kubernetes Affinity rules that are applied to the appropriate DB cluster components.
+
+
+    Kubernetes features three primary types of affinity that play a crucial role in how pods are scheduled and interact within a DB cluster:
+
+    - [Pod affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity){:target="_blank"}: Co-locates pods on the same node or topology domain.
+
+    - [Pod anti-affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity){:target="_blank"}: Ensures pods are scheduled on different nodes or domains.
+
+    - [Node affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity){:target="_blank"}: Schedules pods based on node labels and conditions.
+
+    #### Default configuration for Pod scheduling policies
+
+    
+    In Percona Everest, the default pod scheduling policies are **preset** rules that help ensure optimal placement of database components across a Kubernetes cluster. These predefined Pod scheduling policies come **bundled with every Percona Everest deployment**. Thus, Percona Everest users can utilize these predefined settings without the need to create custom rules for every database cluster they set up.
+
+    ![!image](../images/default_mongo_policy.png)
```

Comment:
in preview mode these images are not shown - please check

Reply:
This will be seen once the brances are merged to main.